static-site/src/sections/sars-cov-2-page.jsx

```diff
@@ -135,17 +135,42 @@ class Index extends React.Component {
   }
 
   componentDidMount() {
+    let filterList;
+    let filterParsed = false;
     try {
-      const filterList = parseDatasetsFilterList(this.state.catalogueDatasets);
-      this.setState({filterList, filterParsed: true});
+      filterList = parseDatasetsFilterList(this.state.catalogueDatasets);
+      filterParsed = true;
     } catch (err) {
       console.error("Error parsing data.", err.message);
     }
+    this.setState({
+      filterList,
+      filterParsed,
+      // assume /ncov/* (or /sars-cov-2/*) is attempting to access dataset
+      // and if this page has been served then the dataset
+      // doesn't exist.
+      // For some reason if this is set in the constructor it breaks the banner.
+      nonExistentDatasetName: this.props["*"]
+    });
+  }
+
+  banner() {
+    if (this.state.nonExistentDatasetName && (this.state.nonExistentDatasetName.length > 0)) {
+      const bannerTitle = `The dataset "nextstrain.org${this.props.location.pathname}" doesn't exist.`;
+      const bannerContents = `Here is the SARS-CoV-2 page, where we have listed featured datasets,
+      narratives, and resources related to SARS-CoV-2. Note that some SARS-CoV-2
+      datasets may not be listed here. For a more comprehensive list of
+      Nextstrain-maintained (including SARS-CoV-2) datasets,
+      check out ${<a href="/pathogens">nextstrain.org/pathogens</a>}.`;
```

Comment:
This is being rendered as 
```
check out [object Object].
```

Reply:
Thanks @jameshadfield! Should be fixed in 8a2dc68.